SwanContents/swancontents/swanclassic/templates/page.html

```diff
@@ -238,6 +264,30 @@
         });
     });
     require(['template']);
+    
+    function _remove_token_from_url() {
```

Comment:
What is this for?

Reply:
This is code copied from upstream. Just aligning our code.